flow/layers/container_layer.h

```diff
@@ -38,6 +47,11 @@ class ContainerLayer : public Layer {
   // For OpacityLayer to restructure to have a single child.
   void ClearChildren() { layers_.clear(); }
 
+  void cacheTotalElevation();
+
+  float elevation_ = 0.0f;
+  float cached_total_elevation_ = 0.0f;
```

Comment:
Nit: maybe we can just name it `total_elevation_`, initialize it to an invalid number (NAN), `FML_DCHECK` that it's a valid number when we're using it so we won't forget to compute it. This style makes it more consistent with how we name and treat the `paint_bounds` and other things that are computed during `Preroll`.

(I'm Ok with any style but it looks strange and hard to reason when there are multiple styles in `Preroll`.)

Reply:
Good idea, done.